simpleprophet/simpleprophet/data.py

```diff
@@ -91,13 +92,18 @@ def get_nondesktop_data(bq_client):
     raw_data = bq_client.query(NONDESKTOP_QUERY).to_dataframe()
     for p in [
         "Fennec Android", "Focus iOS", "Focus Android", "Fennec iOS", "Fenix",
-        "Firefox Lite", "FirefoxForFireTV", "FirefoxConnect"
+        "Firefox Lite", "FirefoxForFireTV", "FirefoxConnect", "Lockwise Android"
     ]:
         data['{} MAU'.format(p)] = raw_data.query("product == @p")[
             ["date", "global_mau"]
         ].rename(
             index=str, columns={"date": "ds", "global_mau": "y"}
         )
+        data['{} tier1 MAU'.format(p)] = raw_data.query("product == @p")[
```

Comment:
Should we style this as "Lockwise Android Tier 1 MAU" or "Lockwise Android Tier-1 MAU" rather than the mixed-case "Lockwise Android tier1 MAU"?

Reply:
And should we update the the names for the global forecasts to be "Lockwise Android Global MAU" to match?